deployment/conf/.templates/deployment.cfg.templ

```diff
@@ -1,5 +1,5 @@
 [idmapping]
-mongo-host={{ default .Env.mongo_host "ci-mongo" }}
+mongo-host={{ default .Env.mongo_host "mongo" }}
```

Comment:
This shouldn't be changed as it might cause backwards incompatible behavior when the service is deployed, if a deployment is relying on the default value

Reply:
👍